src/Components/CroppedThumbnail/CroppedThumbnail.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+import { Image, ImageProps, ImageURISource, StyleSheet } from "react-native";
+
+interface IOptions {
```

Comment:
IOptions name does not give any clue why those options are in a separate interface

Reply:
different interfaces have different types